2025-01-26 16:28:21,707 INFO    MainThread:11990 [wandb_setup.py:_flush():76] Current SDK version is 0.17.7
2025-01-26 16:28:21,708 INFO    MainThread:11990 [wandb_setup.py:_flush():76] Configure stats pid to 11990
2025-01-26 16:28:21,708 INFO    MainThread:11990 [wandb_setup.py:_flush():76] Loading settings from /home/kyeonghyun/.config/wandb/settings
2025-01-26 16:28:21,708 INFO    MainThread:11990 [wandb_setup.py:_flush():76] Loading settings from /home/kyeonghyun/PiFi/Generation/wandb/settings
2025-01-26 16:28:21,708 INFO    MainThread:11990 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2025-01-26 16:28:21,708 INFO    MainThread:11990 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-01-26 16:28:21,708 INFO    MainThread:11990 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Generation/main.py', 'program_abspath': '/home/kyeonghyun/PiFi/Generation/main.py', 'program': 'main.py'}
2025-01-26 16:28:21,708 INFO    MainThread:11990 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-01-26 16:28:21,708 INFO    MainThread:11990 [wandb_init.py:_log_setup():524] Logging user logs to /home/kyeonghyun/PiFi/Generation/wandb/run-20250126_162821-005fj85h/logs/debug.log
2025-01-26 16:28:21,709 INFO    MainThread:11990 [wandb_init.py:_log_setup():525] Logging internal logs to /home/kyeonghyun/PiFi/Generation/wandb/run-20250126_162821-005fj85h/logs/debug-internal.log
2025-01-26 16:28:21,709 INFO    MainThread:11990 [wandb_init.py:init():607] calling init triggers
2025-01-26 16:28:21,709 INFO    MainThread:11990 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {'task': 'translation', 'job': 'training', 'task_dataset': 'multi30k', 'description': 'default', 'method': 'base_llm', 'data_path': '/nas_homes/kyeonghyun/dataset/', 'cache_path': '/nas_homes/kyeonghyun/model', 'preprocess_path': '/nas_homes/kyeonghyun/preprocessed/PiFi_Generation', 'model_path': '/nas_homes/kyeonghyun/model_final/PiFi_Generation', 'checkpoint_path': '/nas_homes/kyeonghyun/model_checkpoint/PiFi_Generation', 'result_path': '/nas_homes/kyeonghyun/results/PiFi_Generation', 'log_path': '/nas_homes/kyeonghyun/tensorbord_log/PiFi_Generation', 'proj_name': 'PiFi_Generation', 'model_type': 't5', 'llm_model': 'llama3.1', 'model_ispretrained': True, 'dropout_rate': 0.2, 'embedding_dims': 768, 'hidden_size': 768, 'num_layers': 3, 'max_seq_len': 100, 'num_transformer_heads': 8, 'num_transformer_layers': 3, 'layer_num': 0, 'optimizer': 'Adam', 'scheduler': 'None', 'num_epochs': 3, 'learning_rate': 5e-05, 'num_workers': 2, 'batch_size': 32, 'weight_decay': 0, 'clip_grad_norm': 5, 'label_smoothing_eps': 0.05, 'early_stopping_patience': 5, 'train_valid_split': 0.2, 'optimize_objective': 'loss', 'test_batch_size': 1, 'device': 'cuda', 'seed': 2023, 'use_tensorboard': False, 'use_wandb': True, 'log_freq': 500, 'src_vocab_size': 32128, 'tgt_vocab_size': 32128}
2025-01-26 16:28:21,709 INFO    MainThread:11990 [wandb_init.py:init():657] starting backend
2025-01-26 16:28:21,709 INFO    MainThread:11990 [wandb_init.py:init():661] setting up manager
2025-01-26 16:28:21,712 INFO    MainThread:11990 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-01-26 16:28:21,714 INFO    MainThread:11990 [wandb_init.py:init():669] backend started and connected
2025-01-26 16:28:21,720 INFO    MainThread:11990 [wandb_init.py:init():764] updated telemetry
2025-01-26 16:28:21,757 INFO    MainThread:11990 [wandb_init.py:init():797] communicating run to backend with 90.0 second timeout
2025-01-26 16:28:22,286 INFO    MainThread:11990 [wandb_run.py:_on_init():2479] communicating current version
2025-01-26 16:28:22,335 INFO    MainThread:11990 [wandb_run.py:_on_init():2488] got version response upgrade_message: "wandb version 0.19.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-01-26 16:28:22,336 INFO    MainThread:11990 [wandb_init.py:init():848] starting run threads in backend
2025-01-26 16:28:24,546 WARNING MainThread:11990 [wandb_init.py:init():1229] interrupted
Traceback (most recent call last):
  File "/home/kyeonghyun/anaconda3/envs/khyun/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1225, in init
    return wi.init()
  File "/home/kyeonghyun/anaconda3/envs/khyun/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 861, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "/home/kyeonghyun/anaconda3/envs/khyun/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/kyeonghyun/anaconda3/envs/khyun/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/kyeonghyun/anaconda3/envs/khyun/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/kyeonghyun/anaconda3/envs/khyun/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/kyeonghyun/anaconda3/envs/khyun/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2025-01-26 16:28:28,141 WARNING MsgRouterThr:11990 [router.py:message_loop():77] message_loop has been closed
